package.json

```diff
@@ -18,32 +18,38 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "mocha",
-    "prepublish": "browserify -o browser.js -e index.js"
```

Comment:
sorry for not proving more details in the commit message. I removed the build script for `browser.js` since, to date, that file has never worked. There was a bug related to lazy-caching that prevented some dependencies from being browserified. I now have a fix for the bug, but it still potentially needs to be fixed in some modules in the tree. 

until I can make the rounds and update everything that uses lazy-cache with the fix, I think it's a good idea to remove that file since we haven't any issues telling us that the browser build is broken in ~9m downloads. If they are, I'd be genuinely curious how they're shimming it. (fwiw, I don't know of any lazy-require module that has solved this before now). 

Thoughts? Were there other potential side-effects I may not have considered?